open-telemetry/Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "open-telemetry",
+    platforms: [.macOS(.v14), .iOS(.v17), .tvOS(.v17)],
+    products: [
+        .executable(name: "App", targets: ["App"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/hummingbird-project/hummingbird.git", from: "2.0.0"),
+        .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.3.0"),
+        .package(url: "https://github.com/swift-otel/swift-otel.git", from: "0.10.0"),
```

Comment:
There may be breaking changes in one of the next minor versions of Swift OTel, so perhaps it's a bit safer to depend on this `upToNextMinor` given the `Package.resolved` file is git-ignored. Alternatively, depending on the `main` branch could make this more explicit.